hoomd/comm.py

```diff
@@ -113,18 +113,18 @@ def __init__(self, x=None, y=None, z=None, nx=None, ny=None, nz=None, linear=Fal
 
             """
             # do a one time update of the cuts to the global values if a global is set
-            if not self.x and self.nx == 0 and hoomd.context.options.nx is not None:
```

Comment:
You don't need to undo these changes. However, no changes are necessary to code that has not been updated to the v3 API.